fairlearn/utils/_common.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+def _get_soft_predictions(estimator, X, predict_method):
+    """Return soft predictions of a classifier using either `predict_proba` \
```

Comment:
for regressors, it returns `predict`; also for classifiers that don't implement `predict_proba` or `decision_function`, it returns `predict`